cmd/fyne/internal/commands/package.go

```diff
@@ -354,12 +355,20 @@ func calculateExeName(sourceDir, osys string) string {
 }
 
 func isValidVersion(ver string) bool {
-	nums := strings.Split(ver, ".")
-	if len(nums) == 0 || len(nums) > 3 {
+	r, _ := regexp.Compile(
```

Comment:
wouldn't it be better / easier to use `semver.IsValid` using the package we already import?
`golang.org/x/mod/semver`

Reply:
Wait. Do I understand you correctly that you want to extend the current spec to also allow version strings with a leading v? The current spec does not allow it.

Reply:
I think Andy means just within `isValidVersion`.